soepy/test/random_init.py

```diff
@@ -130,3 +132,53 @@ def print_dict(init_dict, file_name="test"):
 
     with open("{}.soepy.yml".format(file_name), "w") as outfile:
         yaml.dump(ordered_dict, outfile, explicit_start=True, indent=4)
+
+
+def namedtuple_to_dict(model_params):
```

Comment:
Why do we need this function? Can't we just abandon the use of a dictionary at all?

Reply:
This function is only used in the unit tests. The regression test depends on the print_dict function and there is a unit test checking that this function works properly. I had to add namedtuple_to_dict so that the unit test is still working after the interface change to named tuple.

Reply:
It appears we are still using a `init_dict` repeatedly, please let us remove this everywhere or do we have a reason to keep it around?

Reply:
We will tackle this at a later stage.